src/main/scala/beam/agentsim/agents/vehicles/BeamVehicleType.scala

```diff
@@ -132,11 +132,10 @@ object VehicleCategory {
   case object MediumDutyPassenger extends VehicleCategory
   case object LightDutyTruck extends VehicleCategory
   case object HeavyDutyTruck extends VehicleCategory
-  case object Undefined extends VehicleCategory
 
   def fromString(value: String): VehicleCategory = {
-    Vector(Body, Bike, Car, MediumDutyPassenger, LightDutyTruck, HeavyDutyTruck, Undefined)
+    Vector(Body, Bike, Car, MediumDutyPassenger, LightDutyTruck, HeavyDutyTruck)
       .find(_.toString.equalsIgnoreCase(value))
-      .getOrElse(Undefined)
+      .get
```

Comment:
Another situation where I question if blowing up here is what you want, or if it should return the Option?